vegafusion-python/tests/specs/temporal.json

```diff
@@ -0,0 +1,163 @@
+{
+    "layer": [
+        {
+            "layer": [
+                {
+                    "layer": [
+                        {
+                            "mark": {
+                                "clip": true,
+                                "type": "bar",
+                                "color": "#2266D3",
+                                "tooltip": true
+                            },
+                            "encoding": {
+                                "x": {
+                                    "axis": {
+                                        "grid": false
+                                    },
+                                    "sort": "descending",
+                                    "type": "temporal",
+                                    "field": "datetime",
+                                    "scale": {
```

Comment:
Scale type mismatch for temporal axis

Temporal x-axis shouldn't use linear scale. This will cause rendering issues.
```
-                                    "scale": {
-                                        "type": "linear"
-                                    },
+                                    "scale": {
+                                        "type": "time"
+                                    },
```